### ISSUE TITLE
Sometimes JAWS doesn't work in FF

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#90]
From https://github.com/phetsims/resistance-in-a-wire/issues/132 and phetsims/QA/issues/92.

Do these steps and most JAWS features do not work in any HTML web page.

- Restart Computer
- Enable JAWS
- Enable Firefox

You can tab navigate, and JAWS will read focus, but virtual  cursor and aria-live  do not work. My gut says this is FF problem, so I am going to start with a bug report there.

**Comment by jessegreenberg** [comment]
Hmm, maybe the problem is turning on Firefox while JAWS is still booting up.

**Comment by jessegreenberg** [comment]
Hmm, perhaps the problem happens when turning on FF before JAWS fully boots up. I just waited for JAWS to be ready before opening FF and the problem wasn't there.

**Comment by jessegreenberg** [comment]
I verified that if I wait until  JAWS is fully up and running, then open FF, everything sounds good. Ill report this as an issue to Mozilla

**Comment by jessegreenberg** [comment]
Bug reported here: https://bugzilla.mozilla.org/show_bug.cgi?id=1440083

**Comment by jessegreenberg** [comment]
Pinging @emily-phet and @terracoda so they are aware of this issue. I imagine this could easily happen during interviews or demonstrations, though JAWS users will likely be aware of the problem. As long as JAWS is fully up and running before Firefox is opened this problem shouldn't happen.

**Comment by terracoda** [comment]
@jessegreenberg, thanks for the ping. I will add this as a note in my interview prep documents.

**Comment by jessegreenberg** [comment]
FF responded:

>We know that this is a problem if you start JAWS after Firefox. The workaround
is to run JAWS *before* Firefox and let it finish loading (when it starts
speaking, it's done).

>Note, too, that due to the E10S architecture, our window emulation code for
JAWS that is the culprit here has been changed to acommodate the multi-process
architecture and that JAWS 2018 (Version 19) is the only version supported
going forward, and that a lot of circumstances will have changed once we have
all the perf improvement and fixes for JAWS in place, currently slatted for
JAWS 60. So this bug may be obsoleted by then.

So  it sounds like we will always have launch JAWS before FF, until FF Quantum broadly supports JAWS 19.

Pinging @terracoda, @emily-phet, and @phet-steele so they are aware. For future QA tests will will include this information in the issue. Closing.
